### ISSUE TITLE
`dashd` service installer is missing variable `WEBI_HOST`

### ISSUE THREAD

**Issue by coolaj86** [webinstall/webi-installers#695]
- [x] missing `WEBI_HOST`
- [x] typo: `~/.local/bin/bin/dashd-hd-service-install`

## Problem

```sh
dashd-testnet-service-install /mnt/dashstore/dashcore
```

```text
sudo env PATH="$PATH" serviceman add --system --username app --path "$PATH" --name "dashd-testnet" --force -- dashd  -testnet -usehd -conf="$HOME/.dashcore/dash.conf" -settings="$HOME/.dashcore/settings.json" -walletdir="$HOME/.dashcore/wallets/" -datadir="/mnt/dashstore/dashcore/_data" -blocksdir="/mnt/dashstore/dashcore/_caches"

Installing 'serviceman'...

/home/app/.local/bin/dashd-hd-service-install: 114: WEBI_HOST: parameter not set
env: 'serviceman': No such file or directory
```

## Workaround

```sh
webi serviceman
```

**Comment by coolaj86** [comment]
Also, we should include the openrc init file in the docs in the next PR.

**Comment by coolaj86** [comment]
Also:

```
SOLUTION
        1. Mount a large (50gb+) volume

              sudo mkdir -p /mnt/EXAMPLE
              sudo mount /dev/sdx1 /mnt/EXAMPLE

2. Create a 'dashcore' inside of it

              sudo mkdir -p '/mnt/blah/dashcore'
```

EXAMPLE should be `blah` in this case

**Comment by coolaj86** [comment]
closed via #803
